client/src/components/atoms/Account.module.scss

```diff
@@ -47,7 +47,7 @@
         display: inline-block;
         fill: currentColor;
         margin-right: $spacer / 8;
-        transition: .2s ease-out;
+        transition: 0.2s ease-out;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)